mithril-stm/src/multi_sig.rs

```diff
@@ -748,14 +753,32 @@ mod tests {
         fn serialize_deserialize_vk(seed in any::<u64>()) {
             let mut rng = rand_chacha::ChaCha8Rng::seed_from_u64(seed);
             let sk = SigningKey::gen(&mut rng);
+
             let vk = VerificationKey::from(&sk);
             let vk_bytes = vk.to_bytes();
-            let vk2 = VerificationKey::from_bytes(&vk_bytes).unwrap();
-            assert_eq!(vk, vk2);
+            let result = VerificationKey::from_bytes(&vk_bytes);
+            match result {
+                Ok(vk_from_bytes) => {
+                    assert_eq!(vk, vk_from_bytes);
+                }
+                Err(MultiSignatureError::SerializationError) => {
```

Comment:
I'm wondering why there is this new case that is tested as the `SerializationError` already existed before this PR? I have the same question for all the other modified tests below.